docs/development/best_practices/index.rst

```diff
@@ -0,0 +1,17 @@
+Best Practices for **WaNo** development
+#######################################
+
+- To starting a new **WaNo** for the first time, download one,  copy it into the local repository, and modify it according
+  to the simulation needs. This will speed up the development process.
+
+- The majority of the scientific packages have a considerable number of parameters, which the end-user may, in principle,
+  change. However, only a small subset of parameters is significant for the particular simulation protocol for reoccurring
+  user cases. Thus, it worth spend a bit of time to select this subset of parameters.
```

Comment:
Thus, before writing code, select the subset of parameters significant to the user.